eng/pipelines/coreclr/templates/build-perf-maui-apps-net7.yml

```diff
@@ -119,11 +119,12 @@ steps:
   - script: |
       curl -o NuGet.config 'https://raw.githubusercontent.com/dotnet/maui/main/NuGet.config'
       curl -o dotnet-install.sh 'https://dotnet.microsoft.com/download/dotnet/scripts/v1/dotnet-install.sh'
+      curl -o maui-version.json 'https://maui.blob.core.windows.net/metadata/sdks/7.0.0/rc.1.json'
+      version=$(sed -nr 's/\s*"version": "(.*)"/\1/p' ./maui-version.json)
       chmod -R a+rx .
-      cp $(Build.SourcesDirectory)/src/tests/Common/maui/rollbackfile-net7.json ./rollbackfile-net7.json
-      ./dotnet-install.sh --version 7.0.100-preview.7.22376.5 --install-dir .
+      ./dotnet-install.sh --version $version --install-dir .
       ./dotnet --info
-      ./dotnet workload install maui --from-rollback-file rollbackfile-net7.json --configfile NuGet.config
+      ./dotnet workload install maui --from-rollback-file https://maui.blob.core.windows.net/metadata/rollbacks/7.0.0/rc.1.json --configfile NuGet.config
```

Comment:
@LoopedBard3 is it worth putting these MAUI URLs in yaml variables?

@mattleibow is there an "evergreen" URL we can try? That will automatically change when we get RC 2?

Reply:
https://aka.ms/dotnet/maui/net7.0.json is RC 1 now. And will update once we go RC 2.

Reply:
Is there an aka.ms for the `sdk.json` file?

Reply:
Ok: https://aka.ms/dotnet/sdk/maui/net7.0.json